openhabian-setup.sh

```diff
@@ -80,7 +80,6 @@ if [[ -n "$UNATTENDED" ]]; then
   misc_system_settings
   samba_setup
   clean_config_userpw
-  nodejs_setup
```

Comment:
It is not possible to remove NodeJS from base installation, used by some packages.
Why was this removed?

Reply:
it's called (again) as 1st step in frontail_setup() anyway

Reply:
Hmm you are correct. Let's call the node installation from there.